src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -269,16 +269,13 @@ int get_model_prefer_threads(const int num_streams,
     return model_prefer;
 }
 
-void get_num_streams(const int streams,
-                     const std::shared_ptr<ngraph::Function>& ngraphFunc,
-                     Config& config) {
+void generate_stream_info(const int streams,
+                          const std::shared_ptr<ngraph::Function>& ngraphFunc,
+                          Config& config,
+                          int model_perfer_init) {
```

Comment:
suggest model_perfer_init change to model_prefer_init.  I think this parameter can be set to default value "-1", it will not initialize the variable.

Reply:
Thanks, updated.